08week/dodgeball.js

```diff
@@ -0,0 +1,197 @@
+'use strict'
```

Comment:
I appreciate all the comments throughout the file, nice.
But i am missing the tests or the test plan write up. Either would have been sufficient.